docs/tables.txt

```diff
@@ -1,5 +1,147 @@
-.. _tables:
+.. _tables-browser:
 
 ==============
 Tables Browser
 ==============
+
+The CrateDB admin UI comes with a *tables browser* that allows you to inspect
+and query regular `document tables`_ as well as `BLOB tables`_.
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+Screenshots
+===========
+
+Document Tables
+---------------
+
+Here's what a simple document table looks like:
+
+.. image:: tables-simple-1.png
+
+Here you can see a number of :ref:`table-level datapoints and settings are
+<table-level-info>` listed.
+
+Below this, you will find information about the table :ref:`schema
+<schema-info>`:
+
+.. image:: tables-simple-2.png
+
+BLOB Tables
+-----------
+
+If you have BLOB tables, you can access them by selecting the *Blob Tables* menu
+item on the left-hand sub-navigation menu:
+
+.. image:: tables-blobs-1.png
+
+Like document tables, a number of :ref:`table-level datapoints and settings
+<table-level-info>` are listed first, and then below that, you will find
+information about the table :ref:`schema <schema-info>`:
+
+.. image:: tables-blobs-2.png
+
+Features
+========
+
+.. |nbsp| unicode:: 0xA0
+   :trim:
+
+.. _table-level-info:
+
+Table-Level Information
+-----------------------
+
+The table overview displays the following table-level information:
+
+:Name:
+
+  The name of the table.
+
+:Health:
+
+  The health of the table.
+
+  FIXME: what are the available options
+
+:Configured Replicas:
+
+  The number of configured `replicas`_.
+
+:Configured Shards:
+
+  The `number of configured shards`_
+
+:Started Shards:
+
+  FIXME: what is this?
```

Comment:
@mxm here

Reply:
Started means the shard is assigned to a node and available for querying.